ebssurrogate/scripts/qemu-bootstrap-nix.sh

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+set -o xtrace
+
+if [ $(dpkg --print-architecture) = "amd64" ]; 
+then 
+	ARCH="amd64";
+else
+        ARCH="arm64";
```

Comment:
Nit: inconsistent spaces/tabs throughout the file, but don't need to bother fixing.